module/BeadPoolManifest.py

```diff
@@ -179,6 +184,71 @@ def from_string(source_strand):
             raise ValueError(
                 "Unexpected value for source strand " + source_strand)
 
+
+class IlmnStrand(object):
+    Unknown = 0
+    TOP = 1
+    BOT = 2
+    PLUS = 1
```

Comment:
These should have different values, otherwise to_string behavior below will be undesirable. If necessary, could generalize get_base_calls generic to take a list of report strands (instead of a single value)

Reply:
Changed attribute assignments, can now be handled by get_base_calls_generic, see below.